controller/controller.go

```diff
@@ -232,6 +232,9 @@ func (c *Controller) deleteFunc(ctx context.Context, obj interface{}) {
 // ProcessEvents takes the event channels created by the operatorkit informer
 // and executes the controller's event functions accordingly.
 func (c *Controller) ProcessEvents(ctx context.Context, deleteChan chan watch.Event, updateChan chan watch.Event, errChan chan error) error {
+	c.logger.LogCtx(ctx, "level", "debug", "message", "processing object events")
```

Comment:
The odd thing now is that this log will not contain the loop, wile the last one probably will. I would add this log down right after the loop got added to the context and keep the deferred log as it is. Then the result you observe at operator runtime will be consistent and helpful.